transcom/mymove
# B 17984 update reminder email

## B 17984

## Summary

Merge into main from integration
https://github.com/transcom/mymove/pull/12014

### How to test

1. Create a move with an estimateDepartureDate that is <= 14 days from today, and has no null estimate values(incentive, weight), and set the personal email to one you can access
2. Approve this move as a service counselor
3. Run `aws-vault exec transcom-gov-dev` to get into the subshell
4. Run `go run ./cmd/milmove-tasks send-payment-reminder --email-backend=ses --aws-ses-domain=devlocal.dp3.us  --aws-ses-region=us-gov-west-1`
5. You should get a payment reminder email in a couple minutes max

## Screenshots
<img width="1466" alt="Screenshot 2024-02-21 at 10 36 59 AM" src="https://github.com/transcom/mymove/assets/146969726/7680cf55-464f-44cf-8fd9-9858439ca24f">
